server/model_server/audio_model_server.py

```diff
@@ -49,7 +49,7 @@ def load_soundnet():
 class Api2MslServicer(api2msl_pb2_grpc.Api2MslServicer):
     def __init__(self):
         super().__init__()
-        os.environ['CUDA_VISIBLE_DEVICES'] = '0'
+        os.environ['CUDA_VISIBLE_DEVICES'] = '2'
```

Comment:
Could use a sperate .yml file hardware for configuration

Reply:
E.g:
```yaml
decoder: CPU
model_server:
  audio: CUDA:2
  face: CUDA:0
  text: CUDA:0
  scene: CUDA:1
```

Reply:
add a device_placement.yml file for recording this information
add todo to use this file

We can leave this task later and now focus on testing the system manully.